tests/e2e/testfiles/scenarios.yaml

```diff
@@ -2878,7 +2878,43 @@
         - kubectl get nodes --show-labels
     - name: Validate Node Labels
       run:
-        - /bin/bash modify_zoning_labels.sh validate-zoning
+        - /bin/bash modify_zoning_labels.sh validate-zoning powerflex
+    - name: Remove Node Labels
+      run:
+        - /bin/bash modify_zoning_labels.sh remove-all-zones
+        - kubectl get nodes --show-labels
+
+- scenario: "Install Powermax Driver (Standalone) With Node Zoning"
+  paths:
+    - "testfiles/storage_csm_powermax_secret.yaml"
+  tags:
+    - "zoning"
+    - "powermax"
+  steps:
+    - "Given an environment with k8s or openshift, and CSM operator installed"
+    - "Run [Apply Node Labels]"
+    - "Set up reverse proxy tls secret namespace [powermax]"
+    - "Set up secret with template [testfiles/powermax-templates/powermax-zoning-use-secret-template.yaml] name [powermax-config] in namespace [powermax] for [pmaxUseSecret]"
+    - "Set up creds with template [testfiles/powermax-templates/powermax-array-config.yaml] for [pmaxArrayConfig]"
+    - "Apply custom resource [1]"
+    - "Validate custom resource [1]"
+    - "Validate [powermax] driver from CR [1] is installed"
+    - "Run [Validate Node Labels]"
+    # cleanup
+    - "Enable forceRemoveDriver on CR [1]"
+    - "Delete custom resource [1]"
+    - "Validate [powermax] driver from CR [1] is not installed"
+    - "Run [Remove Node Labels]"
+  customTest:
+    - name: Apply Node Labels
+      run:
+        - kubectl get nodes --show-labels
+        - /bin/bash modify_zoning_labels.sh add zone.topology.kubernetes.io/zone=zoneA zone.topology.kubernetes.io/zone=zoneB
```

Comment:
Shouldn't the labels in `tests/e2e/testfiles/powermax-templates/powermax-zoning-use-secret-template.yaml` be changed to `zone.topology.kub....` as well?

Reply:
Yes, fixed.